tests/unit/amo/components/TestAddon.js

```diff
@@ -773,13 +777,24 @@ describe(__filename, () => {
       getClientCompatibility: () => ({
         compatible: false,
         downloadUrl: 'https://www.seamonkey-project.org',
-        reason: INCOMPATIBLE_NOT_FIREFOX,
+        reason: INCOMPATIBLE_UNDER_MIN_VERSION,
       }),
     });
     expect(root.find(AddonCompatibilityError).prop('downloadUrl'))
       .toEqual('https://www.seamonkey-project.org');
   });
 
+  it('hides banner on non firefox clients and displays firefox download button', () => {
+    const root = shallowRender({
+      getClientCompatibility: () => ({
+        compatible: false,
+        reason: INCOMPATIBLE_NOT_FIREFOX,
+      }),
```

Comment:
I think you can reuse the `getClientCompatibilityFalse` no? Also you should assert for the text on the button.

Reply:
yes - good catch, thanks